docs/changelog/2838.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+A testenv with multiple factors, one of which conflicts with a ``base_python`` setting in ``tox.ini``, will now use the
+correct Python interpreter version - by :user:`stephenfin`.
```

Comment:
It would be the version derived from the factor. If I run `tox -e py310`, I expect to get `python3.10`.

I did have this documented as part of #841 but that seems to have been lost in the rewrite. I can do another PR to (re-)flesh out the documentation. I also need to re-add docs explaining how factors work, which I'd done previously in #1573.